querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -620,6 +663,10 @@ public void serialize(EntityType model, SerializerConfig config,
 
     protected void serialize(EntityType model, Property field, Type type, CodeWriter writer,
             String factoryMethod, String... args) throws IOException {
+
+        String constantFieldName = constantFieldName(field.getEscapedName());
+        writer.publicStaticFinal(new ClassType(String.class), constantFieldName, "\"" + field.getName() + "\"");
```

Comment:
@timowest sure, how would you prefer me passing the option? A code example can help.